h/static/styles/partials-v2/_form-input.scss

```diff
@@ -173,4 +197,11 @@
       max-width: unset;
     }
   }
+
+  @include touch-input {
+    .form-input__label > .form-input__hint-icon {
+      min-height: 44px;
```

Comment:
I'd suggest we define a constant for this (eg. `$touch-target-size`) in `_base.scss` and use it here and in `_btn.scss`